pynest/examples/hpc_benchmark_connectivity.svg

```diff
@@ -0,0 +1,654 @@
+       </g>
+      </g>
+      <g class="com.sun.star.drawing.ClosedBezierShape">
+       <g id="id62">
+        <rect class="BoundingBox" stroke="none" fill="none" x="13525" y="855" width="78" height="234"/>
+        <path fill="rgb(0,0,0)" stroke="none" d="M 13580,875 C 13580,865 13572,855 13560,855 13550,855 13539,863 13539,875 13539,887 13551,895 13560,895 13571,895 13580,886 13580,875 Z M 13527,939 L 13527,952 C 13550,952 13552,954 13552,971 L 13552,1061 C 13552,1076 13548,1076 13526,1076 L 13526,1088 C 13527,1088 13551,1087 13565,1087 13578,1087 13590,1087 13602,1088 L 13602,1076 C 13582,1076 13579,1076 13579,1061 L 13579,936 13527,939 Z"/>
+       </g>
+      </g>
+      <g class="com.sun.star.drawing.ClosedBezierShape">
+       <g id="id63">
+        <rect class="BoundingBox" stroke="none" fill="none" x="13636" y="936" width="187" height="155"/>
+        <path fill="rgb(0,0,0)" stroke="none" d="M 13796,984 C 13796,954 13780,936 13743,936 13715,936 13696,952 13688,970 L 13686,970 13686,936 13636,939 13636,953 C 13660,953 13662,955 13662,972 L 13662,1061 C 13662,1076 13658,1076 13636,1076 L 13636,1089 C 13637,1089 13661,1087 13675,1087 13689,1087 13712,1088 13715,1089 L 13715,1076 C 13693,1076 13690,1076 13690,1061 L 13690,999 C 13690,963 13718,946 13741,946 13765,946 13768,965 13768,983 L 13768,1061 C 13768,1076 13765,1076 13742,1076 L 13742,1089 C 13743,1089 13767,1087 13782,1087 13795,1087 13818,1088 13822,1089 L 13822,1076 C 13799,1076 13796,1076 13796,1061 L 13796,984 Z"/>
+       </g>
+      </g>
+      <g class="com.sun.star.drawing.ClosedBezierShape">
+       <g id="id64">
+        <rect class="BoundingBox" stroke="none" fill="none" x="13902" y="962" width="59" height="149"/>
+        <path fill="rgb(0,0,0)" stroke="none" d="M 13960,1014 C 13960,982 13949,962 13929,962 13911,962 13902,975 13902,988 13902,1001 13911,1015 13929,1015 13937,1015 13942,1011 13948,1008 L 13949,1007 C 13949,1007 13950,1007 13950,1014 13950,1046 13936,1076 13913,1100 13911,1102 13910,1103 13910,1104 13910,1109 13912,1110 13915,1110 13921,1110 13960,1071 13960,1014 Z"/>
+       </g>
+      </g>
+      <g class="com.sun.star.drawing.ClosedBezierShape">
+       <g id="id65">
+        <rect class="BoundingBox" stroke="none" fill="none" x="14089" y="589" width="514" height="515"/>
+        <path fill="rgb(0,0,0)" stroke="none" d="M 14598,607 C 14601,606 14602,602 14602,600 14602,594 14597,590 14592,590 14588,590 14586,591 14585,593 L 14092,1086 C 14090,1088 14089,1091 14089,1093 14089,1099 14093,1103 14099,1103 14101,1103 14104,1102 14106,1100 L 14598,607 Z"/>
+       </g>
+      </g>
```

Comment:
```suggestion
```

Reply:
Thanks @jougs. The diagonal line through the A means that autapses are prohibited which is stated in the description. If needed I can make this more explicit of course.

Reply:
Ah! Thanks for clarifying. I've added a suggestion to the figure caption that would have made things clearer for at least me. Maybe it also helps others?

Reply:
I agree, now it's way better :)